handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -379,12 +379,12 @@ public final SSLEngine newEngine(ByteBufAllocator alloc, String peerHost, int pe
     }
 
     @Override
-    final SslHandler newHandler(ByteBufAllocator alloc, boolean startTls) {
+    public final SslHandler newHandler(ByteBufAllocator alloc, boolean startTls) {
```

Comment:
why `public` instead of `protected`? The base class has this method as `protected` and it doesn't look like these methods need to be `public`.